pyhacrf/pyhacrf.py

```diff
@@ -257,7 +263,8 @@ def forward_backward(self, parameters):
 
     def predict(self, parameters):
         """ Run forward algorithm to find the predicted distribution over classes. """
-        x_dot_parameters = np.dot(self.x, parameters.T)  # Pre-compute the dot product
```

Comment:
For a moment thought it was strange that no tests had to be updated for this tests but I see now that there are a few different `x_dot_parameters`.